addons/mail/static/src/core/common/message.js

```diff
@@ -191,6 +189,15 @@ export class Message extends Component {
                 this.message.body,
             ]
         );
+        this.debouncedPrepareMessageBody = useDebounced(this.prepareMessageBody, 50);
+        useEffect(
+            () => {
+                if (!this.state.isEditing) {
+                    this.debouncedPrepareMessageBody(this.messageBody.el);
+                }
+            },
+            () => [!this.state.isEditing && this.message.body]
+        );
```

Comment:
2nd parameter of `useEffect()` should list the dependencies, so `this.state.isEditing` and `this.message.body`. Avoid using a conditional, because this may result to similar output for 2 different configurations, and you'd expect to re-trigger a `prepareMessageBody()` when it won't because `false` stays `false`